training:
  timesteps: 10000
  algorithm: SAC
  batch_size: 128
  buffer_size: 100000
  model_name: SAC
  
  continue_training: false
  model_path: /data/catkin_ws/src/drl_navigation/models/training_env_no_object/SAC/SAC_200000